### PR TITLE
FPLA-238: Add judge role

### PR DESCRIPTION
**Pull request by swalker125**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/FPLA-238

### Change description ###

Added judicary role.
Role added to AAT and DEMO
UserProfile added to AAT and DEMO

Login: judiciary@mailnesia.com
Password: Password12

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: ccd-definition/AuthorisationCaseType.json [hunk 1]
```diff
@@ -22,5 +22,11 @@
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "UserRole": "caseworker-publiclaw-gatekeeper",
     "CRUD": "CRU"
```
Comment (rebeccabrennan) [inline ccd-definition/AuthorisationCaseType.json]: Should gatekeeper be CRU or R and same for judiciary, surely these should match?

File: ccd-definition/AuthorisationCaseField.json [hunk 2]
```diff
@@ -1083,5 +1083,264 @@
...
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "documents_socialWorkCarePlan_document",
+    "UserRole": "caseworker-publiclaw-judiciary",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "documents_socialWorkOther_border_top",
```
Comment (Jamesnelson117) [inline ccd-definition/AuthorisationCaseField.json]: Investigate wether R access to borders is needed. I'd imagine not

File: ccd-definition/AuthorisationCaseField.json [hunk 3]
```diff
@@ -1083,5 +1083,264 @@
...
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "documents_socialWorkOther",
+    "UserRole": "caseworker-publiclaw-judiciary",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "documents_socialWorkOther_border_bottom",
```
Comment (Jamesnelson117) [inline ccd-definition/AuthorisationCaseField.json]: Same here

File: ccd-definition/AuthorisationCaseField.json [hunk 4]
```diff
@@ -1083,5 +1083,264 @@
...
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "hearingPreferences",
+    "UserRole": "caseworker-publiclaw-judiciary",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "allocationProposal_label",
```
Comment (Jamesnelson117) [inline ccd-definition/AuthorisationCaseField.json]: Same with label

File: kubernetes/configurer/add-users-and-roles.sh [hunk 5]
```diff
@@ -13,7 +13,9 @@ ${dir}/utils/idam-create-caseworker.sh siva@hillingdon.gov.uk caseworker,casewor
 ${dir}/utils/idam-create-caseworker.sh hmcts-admin@example.com caseworker,caseworker-publiclaw,caseworker-publiclaw-courtadmin "(hmcts-admin)"
 ${dir}/utils/idam-create-caseworker.sh cafcass@example.com caseworker,caseworker-publiclaw,caseworker-publiclaw-cafcass "(cafcass)"
 ${dir}/utils/idam-create-caseworker.sh gatekeeper@mailnesia.com caseworker,caseworker-publiclaw,caseworker-publiclaw-gatekeeper "(gatekeeper)"
+${dir}/utils/idam-create-caseworker.sh judiciary@mailnesia.com  caseworker,caseworker-publiclaw,caseworker-publiclaw-judiciary "(judiciary)"
```
Comment (Jamesnelson117) [inline kubernetes/configurer/add-users-and-roles.sh]: Minor spacing issue (code police) ;)

File: ccd-definition/AuthorisationCaseType.json (hunk 1 above)
Comment (swalker125) [reply ccd-definition/AuthorisationCaseType.json]: I think the user just needs to Read the case type. Doesn't update or create currently

File: ccd-definition/AuthorisationCaseField.json (hunk 2 above)
Comment (swalker125) [reply ccd-definition/AuthorisationCaseField.json]: have removed

File: ccd-definition/AuthorisationCaseField.json (hunk 3 above)
Comment (swalker125) [reply ccd-definition/AuthorisationCaseField.json]: have removed

File: ccd-definition/AuthorisationCaseField.json (hunk 4 above)
Comment (swalker125) [reply ccd-definition/AuthorisationCaseField.json]: have removed

File: ccd-definition/AuthorisationCaseField.json [hunk 6]
```diff
@@ -1083,5 +1083,243 @@
...
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "factorsParenting",
+    "UserRole": "caseworker-publiclaw-judiciary",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "uploadDocuments_paragraph_1",
```
Comment (geek-soft) [inline ccd-definition/AuthorisationCaseField.json]: Let's remove it as it is used by upload documents event that is not used by judiciary.

**Pull request merged**